scripts/publish.sh

```diff
@@ -36,3 +36,6 @@ sh -c "./actions-collection/scripts/trivy_scan.sh"
 echo "End: Trivy Scan"
 
 docker push "$IMAGE"
+
+echo "Setting image name output parameter"
+echo "::set-output name=image_name::$IMAGE"
```

Comment:
Make this into env. Look at actions-setup on how to do it.

Reply:
corrected.